Sources/Player/Player.swift

```diff
@@ -117,6 +117,9 @@ public final class Player: ObservableObject, Equatable {
         }
     }
 
+    /// A Boolean value indicating whether playback is currently playing.
+    public var isRunning: Bool { rate != 0 }
```

Comment:
We usually write methods not as one-liners, if we want to I would suggest we do it everywhere in the codebase, through another PR.

```suggestion
    public var isRunning: Bool {
        rate != 0
    }
```